config/routes.rb

```diff
@@ -12,11 +12,11 @@
     get "sign_out", to: "devise/sessions#destroy", as: :destroy_user_session
   end
 
-  post "dataset/:id/approve", to: "datasets#approve", as: :approve_dataset
-  post "dataset/:id/withdraw", to: "datasets#withdraw", as: :withdraw_dataset
-  post "dataset/:id/resubmit", to: "datasets#resubmit", as: :resubmit_dataset
-  get "datasets/:id/datacite", to: "datasets#datacite", as: :dataset_datacite
-  resources :datasets
+  post "work/:id/approve", to: "works#approve", as: :approve_work
+  post "work/:id/withdraw", to: "works#withdraw", as: :withdraw_work
+  post "work/:id/resubmit", to: "works#resubmit", as: :resubmit_work
+  get "works/:id/datacite", to: "works#datacite", as: :dataset_work
+  resources :works
```

Comment:
I am not sure if we want to the URLs to read `/dataset/x` or `/work/x` but for now I made them match the new Work controller to keep things Rails-friendly. We could adjust this if we need to.